apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -211,6 +226,10 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
             access: true,
             type: 'input RegisterMultiPaymentInput { dv: Int!, sender: SenderFieldInput!, groupedReceipts: [RegisterMultiPaymentServiceConsumerInput!]! }',
         },
+        {
+            access: true,
+            type: 'input RegisterMultiPaymentForOneReceiptInput { dv: Int!, sender: SenderFieldInput!, receipt: String!, acquiringIntegrationContext: String! }',
+        },
```

Comment:
If you can, please, change replace `RegisterMultiPaymentReceiptInfoInput` to `BillingReceiptWhereUniqueInput`

Reply:
And also add a `TODO` about changing `consumerId: String!` to `serviceConsumer: ServiceConsumerWhereUniqueInput!`

Reply:
Will do that